Index.html

```diff
@@ -8,60 +8,63 @@
     <title>Renome restaurant</title>
   </head>
 
-  <main>
-    <body>
-      <div class="navigation">
-        <p class="navigation-left">Renome</p>
+  <body>
+    <header>
+      <nav class="navigation">
```

Comment:
if you are not going to have several nav tags then you can remove the class from nav tag and refer to it as nav , just like in the case of header and body